site/docs/hosting/comparison.md

```diff
@@ -0,0 +1,95 @@
+| Scaleway Containers   | Free       | €0.10/100K GB-s, €1.0/100K vCPU-s | [400K GB-s, 200K vCPU-s/mo](https://www.scaleway.com/en/pricing/#serverless-containers)             | :question:         | :question:         | :question:         |                                      |
+| Deno Deploy           | Free       | No paid plans yet                 | [100K req/day, 1000 req/min, 50 ms CPU-time limit](https://deno.com/deploy/docs/pricing-and-limits) | :x:                | :white_check_mark: | :x:                | Beta                                 |
+| Cloudflare Workers    | Free       | $5/10M req                        | [100K req/day, 10 ms CPU-time limit](https://workers.cloudflare.com/)                               | :x:                | :x:                | :white_check_mark: |                                      |
+| Vercel Edge Functions | Free       | $20/mo subscription for 500K      | [100K req/day](https://vercel.com/pricing)                                                          | :question:         | :question:         | :question:         |                                      |
+| Heroku                | Free       | It's complicated                  | [550-1000 h/mo](https://www.heroku.com/pricing)                                                     | :white_check_mark: | :question:         | :question:         | Long startup times, not recommended? |
+| serverless.com        | Free       |                                   |                                                                                                     | :question:         | :question:         | :question:         |                                      |
+| DigitalOcean Apps     | $5         |                                   |                                                                                                     | :question:         | :question:         | :question:         | Not tested                           |
+| Fastly Compute@Edge   |            |                                   |                                                                                                     |                    |                    |                    |                                      |
+
+### VPS
+
+A virtual private server is a virtual machine that you have full control over.
+You can usually access it via [SSH](https://en.wikipedia.org/wiki/Secure_Shell).
+You can install any software there, and you are responsible for system upgrades and so on.
+
+On a VPS, you can run bots using both polling or webhooks.
+
+Check out [the tutorial](./vps.md) on how to host grammY bots on a VPS.
+
+| Name          | Min. price | Ping to API                               | Cheapest option                    |
```

Comment:
```suggestion
| Name          | Min. price | Ping to Telegram API                               | Cheapest option                    |
```

I think this is `Ping to Telegram API`? If it is, should add it.

Reply:
To me, it makes sense that you naturally thought that it is Telegram's API and not something else.

Reply:
It can actually be confusing.
To me, it can be `Ping to Telegram API` or `Somewhere Ping to the Hosting Providers API`.
It won't hurt to make it clear.

Reply:
We should know the boss' idea, @KnorpelSenf.

Reply:
It's not the ping to the Telegram API. It's the ping to the Bot API if you're using the hosted one, rather than running your own. We can write `Ping to Bot API`.

Reply:
So it's somewhere ping to the bot hosted vps?

```
[Bot]
[VPS] <-> [Somewhere]
```

Reply:
[VPS] <-> api.telegram.org